datawig/_hpo.py

```diff
@@ -65,12 +67,12 @@ def __init__(self):
         default_hps = dict()
         default_hps['global'] = {}
         default_hps['global']['learning_rate'] = [3e-4]
-        default_hps['global']['weight_decay'] = [0, 1e-7]
-        default_hps['global']['num_epochs'] = [100]
+        default_hps['global']['weight_decay'] = [1e-7]
+        default_hps['global']['num_epochs'] = [10]
```

Comment:
Why not leave at 100? I would thing it's better to have early stopping as expected reason for termination rather than limiting to 10 epochs.